mgmtfn/k8splugin/driver.go

```diff
@@ -448,7 +476,7 @@ func addPod(r *http.Request) (interface{}, error) {
 	}
 
 	// Set default gateway
-	err = setDefGw(pid, gw, gwIntf)
+	err = setDefGw(pid, gw, ep.IPv6Gateway, gwIntf)
```

Comment:
I believe this is the response format expected from CNI https://github.com/contiv/netplugin/blob/master/mgmtfn/k8splugin/cniapi/api.go#L42-L48. Right now, it doesn't capture IPv6. Can the CNI response object be modified to accommodate more fields (IPv6 here)? Is it supported?